HIP/hip-1086.md

```diff
@@ -89,6 +89,9 @@ The pricing will be determined by the usual gas price logic in the EVM world.
 This HIP only permits the `EthereumTransaction` to be "jumbo" transactions. Any other type of transactions exceeding 6KB should 
 be rejected unless specifically allowed by a future HIP.
 
+This HIP only applies to single transactions. Jumbo Ethereumtransactions will not be allowed to be executed on Batch transactions,  
```

Comment:
We could add a link so someone that comes to this HIP first can find more details on Jumbo Transactions if needed.
```suggestion
This HIP only applies to single transactions. [Jumbo Ethereumtransactions](https://github.com/hiero-ledger/hiero-improvement-proposals/pull/1086) will not be allowed to be executed on Batch transactions,  
```

Reply:
Questions

- why are we linking Jumbo to Jumbo? Your suggestion is to link to the same doc :) 
- If we words out the 6kk, which i thought to remove too, we need to remove the other mentions on the HIP

Reply:
whoops. I mixed up HIPs. I thought this was the Batch Transactions HIP being update.
No-op 😆

Reply:
Yes, the use of Hedera a configured numbers in many HIPs is something we might want to revisit.
Hiero will have configs and maybe default values. But networks like Hedera that deploy Hiero can set what they like.
We just want to be clear going forward in new HIPs or currently not completed ones that there is a config and the value is set at the network level and can change according to that networks rules.